### ISSUE TITLE
When TOC hide is used in page, sidebar TOC is not displayed. 

### ISSUE THREAD

**Issue by jaredzimmerman** [prtksxna/persistent-toc#16]
Perhaps in-page TOC should be removed completely when gadget is enabled?

**Comment by prtksxna** [comment]
> When TOC hide is used in page, sidebar TOC is not displayed

Even though this is a bug, I'd call it a feature :smile: 

> Perhaps in-page TOC should be removed completely when gadget is enabled?

I am not too sure about this. @quiddity-wp what do you think?

**Comment by quiddity-wp** [comment]
We couldn't remove the default ToC, because that would mean any ToC is inaccessible until the user has scrolled down enough to trigger the sidebar ToC to appear.
I like the current configuration.

**Comment by skanduru** [comment]
used with the skin 'vector-2022'  of mediawiki in a file of its own common.js at the top-level directory.
The collapsible feature of vector-2022 doesn't work anymore. Instead the main frame does a sideways movement. Nice to have the collapsibility feature back.

Secondly persistent toc appears only after moving  down the page until the admin links of "Main Page" and "Tools" are no more visible. 

At that point sometimes, the persistent-toc and the main frame page flickers and becomes hard to read

Nice to fix the above 3 issues.

Finally is it possible to make the TOC collapsible hiding the sublevels when the parent level is clicked ?
Thanks
